docker/dev.yml

```diff
@@ -1,20 +1,37 @@
 ﻿version: "3.7"
+
+x-backend-conf: &backend-conf
+  image: openfoodfacts-server/backend:dev
+  build:
+    context: .
+    dockerfile: Dockerfile
+    # align user id
+    args:
+      USER_UID: ${USER_UID:-1000}
+      USER_GID: ${USER_GID:-1000}
+  volumes:
+    # mount local folder for reload on dev changes.
+    # Note that this means /opt/product-opener/html/images/product wont be connected to product_images volume
+    # Sadly, there is no sane way to do this, while retaining compatibility with prod
+    # (which requires images/product to be empty in git)
+    - ./:/opt/product-opener
+    # we use this for debugging, from times to times
+    - ./debug:/mnt/podata/debug/
+
+
 services:
-  backend:
-    image: openfoodfacts-server/backend:dev
-    build:
-      context: .
-      dockerfile: Dockerfile
-      target: runnable
-    volumes:
-      - ./:/opt/product-opener # mount local folder for reload on dev changes
+  backend: *backend-conf
+  incron: *backend-conf
```

Comment:
this doesn't match what's in the docker-compose.yml setup where we override the `command` field for `incron` service.

Reply:
It's ok as we do not redefine the `command:` data in backend-conf.